miso-web/src/main/webapp/scripts/project_ajax.js

```diff
@@ -193,7 +192,7 @@ Project.ui = {
             "bJQueryUI": true,
             "iDisplayLength": 25,
             "aaSorting": [
-              [0, "desc"]
+              [0, "asc"]
```

Comment:
Fair. It looked really weird the other way, but I guess we sort everything else by date.